atlasdb-impl-shared/build.gradle

```diff
@@ -35,5 +35,5 @@ dependencies {
   testCompile group: 'org.mockito', name: 'mockito-core'
   testCompile group: 'org.hamcrest', name: 'hamcrest-library'
 
-  testRuntime group: 'ch.qos.logback', name: 'logback-classic'
+  testCompile group: 'uk.org.lidalia', name: 'slf4j-test', version: '1.1.0'
```

Comment:
logback was unused, slf4j-test is used in other projects and reused here. Good point about the version.